app/assets/javascripts/student_profile_v2/attendance_details.js

```diff
@@ -7,6 +7,31 @@
   var ProfileChart = window.shared.ProfileChart;
   var Scales = window.shared.Scales;
 
+  var styles = {
+    box: {
+      border: '1px solid #eee',
+      padding:'15px',
+      marginTop: '10px',
+      marginBottom: '10px',
```

Comment:
In React styles, it will interpret numbers for properties like `margin`, etc. as pixels so you can use plain numbers here like `10`.